assets/src/edit-story/components/panels/stylePreset/presets.js

```diff
@@ -187,7 +152,7 @@ Presets.propTypes = {
   handleOnClick: PropTypes.func.isRequired,
   isEditMode: PropTypes.bool.isRequired,
   isText: PropTypes.bool.isRequired,
-  textContent: PropTypes.string,
+  isBackground: PropTypes.bool.isRequired,
```

Comment:
A better name would be `allowsOpacity` or similar, as that's the relevant distinction.